server/src/storage/datastore.ts

```diff
@@ -33,4 +33,21 @@ const getWithId = async (kind: string, id: string) => {
   };
 };
 
-export {getWithId};
+/**
+ * A Datastore wrapper that gets all entities of a specified Kind.
+ * @param kind The Kind that is being queried
+ */
+const getAllWithId = async (kind: string) => {
+  const query = datastore.createQuery(kind);
+  const [res] = await datastore.runQuery(query);
+  const resWithId = res.map(item => {
```

Comment:
I think you can refactor out this function since it's used in `getWithId` also.

Reply:
ok! was thinking of doing that too!